src/OrchardCore.Modules/OrchardCore.Email.Smtp/Services/SmtpOptionsConfiguration.cs

```diff
@@ -25,9 +25,9 @@ public class SmtpOptionsConfiguration : IConfigureOptions<SmtpOptions>
 
     public void Configure(SmtpOptions options)
     {
-        var settings = _siteService.GetSiteSettingsAsync()
-            .GetAwaiter().GetResult()
-            .As<SmtpSettings>();
+        var settings = _siteService.GetSettingsAsync<SmtpSettings>()
+            .GetAwaiter()
+            .GetResult();
```

Comment:
As I said in another PR the VS adds 4 spaces when you move the method call in the next line

/cc @hyzx86